docs/deployment-key-rotation.md

```diff
@@ -3,25 +3,26 @@ Martha Deployment Key Rotation
+3. Disable all existing key(s) on [this list](https://console.cloud.google.com/iam-admin/serviceaccounts/details/107440104000315564432/keys?project=broad-dsde-prod). Using your @firecloud.org account:
+```
+gcloud config set project broad-dsde-prod
+gcloud iam service-accounts keys disable [key_id] --iam-account=cloud-functions-account@broad-dsde-prod.iam.gserviceaccount.com
+```
+4. Using your `@firecloud.org` account, [create a new key for the `cloud-functions-account@broad-dsde-prod.iam.gserviceaccount.com` service account](https://console.cloud.google.com/iam-admin/serviceaccounts/details/107440104000315564432/keys?project=broad-dsde-prod).
    1. Click `[ADD KEY]` > `Create new key`
    2. Select the JSON key type
    3. Click `[CREATE]` and make sure the key `.json` file downloads to your computer
-4. Write this new key to vault.
+5. Write this new key to vault.
 ```
-vault write secret/dsde/martha/prod/deploy-account.json @[key .json file]
-```
-5. Disable or delete the existing key (id found on above page). Using your `@firecloud.org` account:
-```
-gcloud config set project broad-dsde-prod
-gcloud iam service-accounts keys disable [key_id] --iam-account=cloud-functions-account@broad-dsde-prod.iam.gserviceaccount.com
+vault write secret/dsde/martha/prod/deploy-account.json @/path/to/key.json
 ```
 6. Go to the [Jenkins prod manual deploy project](https://fcprod-jenkins.dsp-techops.broadinstitute.org/job/martha-manual-deploy/) and re-run the last Martha prod job. This will re-deploy the current version of Martha, verifying that the new service account key works.
 7. Again, [run the full manual test on the prod environment](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng) to make sure Martha still works.
-8. Clone the Jira ticket you've been working on, without cloning links or sprint fields. Update the due date to be 12 weeks (84 days) from today (which gives about a week to respond when the time comes).
-9. Add a slack reminder in `#dsp-batch-private` to trigger 84 days from today. Make sure to include links to these instructions and the Jira ticket you just created.
+8. Now that we know it works, delete the disabled key(s) from [the list](https://console.cloud.google.com/iam-admin/serviceaccounts/details/107440104000315564432/keys?project=broad-dsde-prod) by selecting the trash can icon. Only the one created today should remain.
+9. Clone the Jira ticket you've been working on, without cloning links or sprint fields. Update the due date to be 12 weeks (84 days) from today (which gives about a week to respond when the time comes).
+10. Add a slack reminder in `#dsp-batch-private` to trigger 84 days from today. Make sure to include links to these instructions and the Jira ticket you just created.
```

Comment:
One more Slack channel to change!

Reply:
Thank you!

![facepalm-head](https://user-images.githubusercontent.com/1087943/199318626-95f9b904-8a8d-455c-8699-e6b35a715f57.jpg)